adminSiteClient/DatasetEditPage.tsx

```diff
@@ -270,17 +260,6 @@ class DatasetEditor extends Component<{ dataset: DatasetPageData }> {
                             Explore in Wizard
                         </Button>
                     </a>
-                    {/* View on GitHub link (old) */}
-                    {!isBulkImport && !dataset.isPrivate && (
-                        <a
-                            href={this.gitHistoryUrl}
-                            target="_blank"
-                            className="btn btn-secondary"
-                            rel="noopener"
-                        >
-                            <FontAwesomeIcon icon={faGithub} /> View on GitHub
-                        </a>
-                    )}
```

Comment:
I think it's fine to get rid of this now, even though it is currently still present for very old datasets (e.g. [this one](https://admin.owid.io/admin/datasets/2878)). Is it still useful?

Reply:
> Is it still useful?

Well... no.